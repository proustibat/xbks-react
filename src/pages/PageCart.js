import React from 'react';
import Cart from '../components/cart/Cart';

const PageCart = () => (
    <Cart />
);

export default PageCart;
