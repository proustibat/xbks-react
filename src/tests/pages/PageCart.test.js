import React from 'react';
import { shallow } from 'enzyme';
import PageCart from '../../pages/PageCart';

test( 'Should render cart page correctly', () => {
    const wrapper = shallow( <PageCart /> );
    expect( wrapper ).toMatchSnapshot();
} );
