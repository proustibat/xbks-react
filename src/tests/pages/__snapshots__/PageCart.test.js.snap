// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render cart page correctly 1`] = `<Cart />`;
